web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
@bealdav It's a question of homogeneity in the files layout, no matter the number of lines. Please split the file and name each file with the model name.